core/src/main/resources/reference.conf

```diff
@@ -295,6 +295,9 @@ filesystems {
       config {
         martha {
           url = "https://martha-url-here"
+          # When true any DrsPath will be internally converted to another compatible Path such as a GcsPath
```

Comment:
should there be a "when possible" in that sentence?

Reply:
updated